docs/contributing/marketplace.md

```diff
@@ -0,0 +1,29 @@
+---
+id: marketplace
+title: Contributing From Cortex XSOAR Marketplace
```

Comment:
I don't think we need `Marketplace` in the title. The process is actually contributing from Cortex XSOAR to the Marketplace.
```suggestion
title: Contributing From Cortex XSOAR
```